cli/cmd/verify.go

```diff
@@ -131,18 +148,25 @@ func parseVerifyFlags(cmd *cobra.Command) (*verifyFlags, error) {
 		return nil, err
 	}
 
+	if workspaceDir != "" {
+		// Prepend default path with workspaceDir
+		if !cmd.Flags().Changed("manifest") {
+			manifestPath = filepath.Join(workspaceDir, manifestFilename)
+		}
+	}
+
 	return &verifyFlags{
+		manifestPath: manifestPath,
 		coordinator:  coordinator,
 		workspaceDir: workspaceDir,
 		policy:       policy,
 	}, nil
 }
 
-func newCoordinatorValidateOptsGen(hostData []byte) *snp.StaticValidateOptsGenerator {
-	defaultManifest := manifest.Default()
-	trustedMeasurement, err := defaultManifest.ReferenceValues.TrustedMeasurement.Bytes()
+func newCoordinatorValidateOptsGen(mnfst manifest.Manifest, hostData []byte) (*snp.StaticValidateOptsGenerator, error) {
```

Comment:
This is now very similar to `authority.SNPValidateOpts`, we should imho share that code.

Reply:
I moved the shared code to the `manifest` package, to generate a `validate.Options` struct from the manifest.